components/hash/node/index.mjs

```diff
@@ -0,0 +1,9 @@
+import { createHash } from "node:crypto";
+
+export const hashFile = ({ url, content }) => {
+  const hash = createHash("sha256");
+  hash.update(url, "utf8");
+  hash.update("\0", "utf8");
+  hash.update(content, "utf8");
```

Comment:
I think utf8 is the default and doesn't need to be specified.

Reply:
I know, but I like being explicit about it. I consistently use provide an explicit `"utf8"` encoding to `node:fs` methods...